src/_includes/banner.html

```diff
@@ -1,8 +1,6 @@
 {% comment %}Drop the div .alert classes to revert to the default banner styling.{% endcomment -%}
 <div class="banner alert alert-info">
   <p class="banner__text">
-    <a href="https://medium.com/dartlang/dart-2-stable-and-the-dart-web-platform-3775d5f8eac7" target="_blank" rel="noopener"><b>Dart 2</b> is now stable!</a>
-    <a href="/dart-2#migration">Migrate your code</a> or
-    <a href="/install">install Dart</a>.
+    TEXT GOES HERE
```

Comment:
Maybe put this message in a comment, in case it ever is displayed by mistake :)

Reply:
I did that and also applied your suggestions from #1138.